resources/event-bus/values.yaml

```diff
@@ -6,8 +6,8 @@ global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
   event_bus:
-    dir: develop/
-    version: 7fb33887
+    dir: pr/
+    version: PR-6266
```

Comment:
We need a real sha here as soon as the [event-bus changes](https://github.com/kyma-project/kyma/pull/6266) are mereged. Otherwise it will block the next Kyma release.